web/source/keyboards/kmwkeyboards.ts

```diff
@@ -1181,7 +1181,7 @@ namespace com.keyman.keyboards {
      * @param {string} x keyboard name string
      * 
```

Comment:
Can you add a comment how `force` gets used? The commit message said it's for a unit test, but I don't see the usage.

Reply:
Good catch; it's used now by the new tests, and I added the new comment bit from kmwbase.ts here as well.